firehydrant/types.go

```diff
@@ -136,6 +136,43 @@ type ServicesResponse struct {
 	Services []ServiceResponse `json:"data"`
 }
 
+// UserResponse is the payload for a user
+// URL: GET https://api.firehydrant.io/v1/users
+type GetUserParams struct {
+	Query string `url:"query,omitempty"`
+}
+
+type User struct {
+	ID          string    `json:"id"`
+	Name        string    `json:"name"`
+	Email       string    `json:"email"`
+	SlackLinked string    `json:"slack_linked?"`
```

Comment:
this will need to be a bool
```suggestion
	SlackLinked bool    `json:"slack_linked?"`
```